test/helpers/kubectl.go

```diff
@@ -49,9 +50,18 @@ type Kubectl struct {
 }
 
 // CreateKubectl initializes a Kubectl helper with the provided vmName and log
+// It marks the test as Fail if cannot get the ssh meta information or cannot
+// execute a `ls` on the virtual machine.
 func CreateKubectl(vmName string, log *logrus.Entry) *Kubectl {
 	node := GetVagrantSSHMeta(vmName)
 	if node == nil {
+		ginkgo.Fail(fmt.Sprintf("Cannot connect to vmName  '%s'", vmName), 1)
+		return nil
+	}
+	res := node.Exec("ls /tmp/")
```

Comment:
What is the reason for checking the result of this command? It seems arbitrary

Reply:
Hi @ianvernon 

I saw a few times that the ssh meta information was correct but can't execute commands, and all commands start to fail. 

That `ls` make sure that can ssh into and run commands without problems, and prevent further issues.

Reply:
OK, so this is a basic sanity check. Can you add a comment equivalent to your above explanation to each place where you do this `ls` check?